apps/public/src/app/major-events/registration/standard/subscription-form-flow.css

```
@@ -0,0 +1,143 @@
+
+  .flow-header h2 {
+    font: var(--mat-sys-title-large);
+  }
+
+  .form-step,
+  .agreement-step {
+    margin: 0;
+    padding: 16px 0 4px;
+  }
+
+  :host ::ng-deep .mat-vertical-content-container {
```

Stylelint (17.14.0)

[error] Unknown pseudo-element selector "::ng-deep" (selector-pseudo-element-no-unknown)

(selector-pseudo-element-no-unknown)